### PR TITLE
COV-195: add fake data gen scripts

### PR DESCRIPTION
**Pull request by jason-gerard**
JIRA: https://soen-390-covid-tracker.atlassian.net/browse/COV-195

## changes
- Add faker js to generate data
- Add fixtures from generated seed data to test actual values
- Move default date from db schemas to services

### CODE REVIEW

File: server/db/schemas/statuses.sql [hunk 1]
```diff
@@ -2,7 +2,7 @@ CREATE TABLE statuses (
     status_id SERIAL,
     patient_id INT NOT NULL,
     status JSON NOT NULL,
-    created_on TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT current_timestamp,
+    created_on TIMESTAMP WITH TIME ZONE NOT NULL,
```
Comment (jason-gerard) [inline server/db/schemas/statuses.sql:5]: We should define the created on dates in the service layer so we have more control over them

Comment (Andre-Ibrahim) [reply server/db/schemas/statuses.sql:5]: I was thinking the current time to be the fallback value if we don't define a time up to you if you want to remove it tho

File: server/db/seed_data/sample_symptoms.ts [hunk 2]
```diff
@@ -0,0 +1,153 @@
+export const sampleSymptoms = [
+    "Suddenly she felt a sharp pain in her neck, shoulder, waist, arm, leg, stomach.",
```
Comment (Andre-Ibrahim) [inline server/db/seed_data/sample_symptoms.ts:2]: so that's what took you so long ;)

**Review by Andre-Ibrahim — APPROVED** [review]
looks good!

File: server/db/schemas/statuses.sql (hunk 1 above)
Comment (jason-gerard) [reply server/db/schemas/statuses.sql:5]: We could do that but its best to keep a single source of truth for date generation. basically we just make sure its consistent everywhere so no subtle bugs arrive

**Pull request merged**